app/src/components/main/navigationTabs/navigationTabs.scss

```diff
@@ -17,7 +17,7 @@
 .navigation-tabs {
   width: 100%;
   box-sizing: border-box;
-  padding: 5px 15px 15px 15px;
+  padding: 5px 45px 15px 45px;
```

Comment:
I think it is better not to change the padding directly in the general-purpose component.
It is used in several other places, and padding there 5px 15px 15px 15px;